meteor/client/ui/FloatingInspectors/NoraFloatingInspector.tsx

```diff
@@ -205,6 +206,9 @@ export class NoraPreviewRenderer extends React.Component<{}, IStateHeader> {
 	}
 
 	render() {
+		const stepContent = this.state.noraContent?.payload?.step
+		const isMultiStep = !!this.state.noraContent?.payload?.step?.enabled
```

Comment:
`this.state.noraContent?.payload?.step?.enabled === true` gets rid of the double negation and is more explicit on what the actual test is (enabled must be true)